infra/terraform/test-org/github/protection.tf

```diff
@@ -135,6 +135,22 @@ module "codeowners_gcp" {
   repos_map = local.gcp_modules_map
 }
 
+module "lint_yaml_tgm" {
```

Comment:
It would be good to have an opt out flag to manage this in repo if any repos additional setup/params

Reply:
add `disable_lint_yaml` to opt out of managed `lint.yaml`